tests/smoke/events.js

```diff
@@ -57,7 +57,7 @@
 				});
```

Comment:
idea:  can we move the "resumeAfter" listener registration out of `bot.setData`'s callback?

Probable timing issue where the resume listener is registered after the data has already been set.  I don't know why it is nested here.  Probably misunderstanding on my part, or it used to work, because we were manually starting the spellcheck